wagtailio/project_styleguide/templates/patterns/pages/search_results/search_results.html

```diff
@@ -23,11 +26,21 @@ <h1 class="hero__heading">Search results</h1>
 
     <div class="grid">
         <div class="search-page__results">
+            {% get_search_promotions search_query as search_picks %}
+            {% if search_picks %}
+                {% for pick in search_picks %}
+                    {% with page=pick.page.specific %}
+                        <div class="search-result">
+                            <a class="search-result__title" href="{{ page.url }}"><h3>{{ page.title }}</h3></a>
+                            <p>{% firstof page.listing_intro page.introduction %}</p>
+                        </div>
+                    {% endwith %}
+                {% endfor %}
+            {% endif %}
+
             {% if search_results %}
-                {% with count=search_results.count %}
-                    <p class="intro-small">{{ count }} Result{{ count|pluralize }} for{% if search_query %}
-                        {{ search_query }}{% endif %}</p>
-                {% endwith %}
+                <p class="intro-small">{{ result_count }} Result{{ result_count|pluralize }} for{% if search_query %}
+                    {{ search_query }}{% endif %}</p>
 
                 {% for result in search_results %}
                     {% with page=result.specific %}
```

Comment:
somehow didn't notice it the first time, but this (and `pick.page.specific`) is inneficient.

Best to move this to the queryset. (relevant old release note: https://docs.wagtail.org/en/stable/releases/1.1.html#specific-method-on-pagequeryset)

Reply:
Thanks @zerolab I've made some changes in f39e032f